llama/runner/README.md

```diff
@@ -19,3 +19,39 @@ curl -X POST -H "Content-Type: application/json" -d '{"prompt": "hi"}' http://lo
 ```
 curl -X POST -H "Content-Type: application/json" -d '{"prompt": "turn me into an embedding"}' http://localhost:8080/embeddings
 ```
+
+## Building
+
+To build the `runner` executable, you first need to build the parent `llama` module:
+
+```sh
+(cd .. && go build .)
+```
+
+From there, you can build this executable:
+
+```sh
+go build .
+```
+
+### macOS Version Issues
+
+If building for a version of macOS above `11.3`, you may run into version conflicts when building the `runner` since the `llama` module explicitly targets `11.3`:
```

Comment:
Thanks for pointing out the issue. Daniel sent out a PR to fix this:
https://github.com/ollama/ollama/pull/7203

Given that this is an issue on `main`, I think it's best to not include fixes to it here (both in the README and the `llama.go` build flags). That way we can keep the changes clearly identifiable and not cause merge conflicts.